src/index.ts

```diff
@@ -130,7 +130,7 @@ export default class ThingsLogbookPlugin extends Plugin {
     );
 
     const daysToTasks: Record<string, ITask[]> = groupBy(
-      tasks.filter((task) => task.stopDate),
+      tasks.filter((task) => task.stopDate).map((task) => task),
```

Comment:
```suggestion
      tasks.filter((task) => task.stopDate),
```

Reply:
Ha-ha! Missed that one! 
I wanted to replace notes with empty strings here 😅